src/Catty/Extension&Delegate&Protocol/Extensions/Notification/NotificationExtension.swift

```diff
@@ -33,6 +33,7 @@ extension Notification.Name {
     static var projectInvalidXml: Notification.Name { .init(rawValue: NotificationName.projectInvalidXml) }
     static var projectFetchDetailsFailure: Notification.Name { .init(rawValue: NotificationName.projectFetchDetailsFailure) }
     static var settingsCrashReportingChanged: Notification.Name { .init(rawValue: NotificationName.settingsCrashReportingChanged) }
+    static var projectSearchFailure: Notification.Name { .init(rawValue: NotificationName.projectSearchFailure) }
```

Comment:
Minor suggestion: please cluster all project-related notifications (e.g. move it above `settingsCrashReportingChanged`).